File: project2/Team1/spring/src/main/java/bitc/full502/spring/dto/PostDto.java
package bitc.full502.spring.dto;

import lombok.*;
import java.time.LocalDateTime;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class PostDto {
    private Long id;
    private String title;
    private String content;
    private String imgUrl;
    private Long lookCount;
    private long likeCount;
    private String author;
    private Boolean liked;           // 로그인 사용자가 좋아요 눌렀는지
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: project2/Team1/android/app/src/main/java/bitc/fullstack502/android_studio/network/dto/PostDto.kt
package bitc.fullstack502.android_studio.network.dto

data class PostDto(
    val id: Long,
    val title: String,
    val content: String,
    val imgUrl: String?,
    val author: String,
    val likeCount: Int,
    val lookCount: Int,
    val liked: Boolean? = false
)

data class PagePostDto(
    val content: List<PostDto>,
    val number: Int,
    val totalPages: Int,
    val totalElements: Long
)
